app/page.cy.tsx

```diff
@@ -19,4 +19,23 @@ describe("Home", () => {
       "Create new keys and Sign Up"
     );
   });
+
+  it("should show the keys when clicking the create new keys link", () => {
```

Comment:
This and every `it` has to be capitalized.

Reply:
I have capitalized the descriptions of the tests.